docs/en_US/TrainingService/RemoteMachineMode.rst

```diff
@@ -144,7 +144,7 @@ Files in ``codeDir`` will be uploaded to remote machines automatically. You can
 Configure python environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, commands and scripts will be executed in the default environment in remote machine. If there are multiple python virtual environments in your remote machine, and you want to run experiments in a specific environment, then use **preCommand** to specify a python environment on your remote machine. 
+By default, commands and scripts will be executed in the default environment in remote machine. If there are multiple python virtual environments in your remote machine, and you want to run experiments in a specific environment, then use **pythonPath** to specify a python environment on your remote machine. 
```

Comment:
How does it work? If I execute a command line tool that uses python, for example, nnictl, lightgbm, horovodrun, will it work?

Reply:
When you set this pythonPath, it will be added to the front of env PATH in current terminal, which means when you run any tool that calls python or run python directly from the terminal, this python your provided the path will be chosen first. 
But it only works for current terminal which your set the pythonPath at first.